ginivision/src/main/java/net/gini/android/vision/camera/CameraFragmentImpl.java

```diff
@@ -50,6 +54,8 @@
 
 class CameraFragmentImpl implements CameraFragmentInterface {
 
+    public static final String GV_SHARED_PREFS = "GV_SHARED_PREFS";
```

Comment:
You can use the `OncePerInstallEventStore` with a `OncePerInstallEvent`.

Reply:
Thought of it as well, but then `OncePerInstallEventStore` & `OncePerInstallEvent` has to be public. If this is fine for you I can change it.

Reply:
Yeah it can be public, we flag non-public API public classes (Java packages 🤢) with @exclude in the Javadoc and won't be shown in our Javadocs...